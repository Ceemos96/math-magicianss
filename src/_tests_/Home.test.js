import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Testing the Homepage', () => {
  it('Home is rendered', () => {
    const home = renderer
      .create(<Home />)
      .toJSON();
    expect(home).toMatchSnapshot();
  });
});
