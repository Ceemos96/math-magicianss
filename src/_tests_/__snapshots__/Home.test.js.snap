// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Homepage Home is rendered 1`] = `
<div
  className="home-wrapperr"
>
  <h3>
    Welcome to Math Magicians!!
  </h3>
  <p>
    A mathematician is a person who can find analogies between theorems; a better mathematician is one who can see analogies between proofs and the best mathematician can notice analogies between theories.But above all I wish to designate the following as the most important among the numerous questions which can be asked with regard to the axioms: To prove that they are not contradictory, that is, that a definite number of logical steps based upon them can never lead to contradictory results.
  </p>
  <p>
    The elegance of a mathematical theorem is directly proportional to the number of independent ideas one can see in the theorem and inversely proportional to the effort it takes to see them.
  </p>
</div>
`;
